include/hex_quadrature.tcc

```diff
@@ -0,0 +1,243 @@
+// Getting the quadratures for Tnquadratures = 1
+template <>
+inline Eigen::MatrixXd mpm::HexahedronQuadrature<3, 1>::quadratures() {
+  Eigen::Matrix<double, 1, 3> quadratures;
+    quadratures(0, 0) = 0.;
+    quadratures(0, 1) = 0.;
+    quadratures(0, 2) = 0.;
+
+  return quadratures;
+}
+
+// Getting the weights for Tnquadratures = 1
+template <>
+inline Eigen::VectorXd mpm::HexahedronQuadrature<3, 1>::weights() {
+  Eigen::VectorXd weights(1);
+    weights(0) = 8.;
+
+  return weights;
+}
+
+// Getting the quadratures for Tnquadratures = 8
+template <>
+inline Eigen::MatrixXd mpm::HexahedronQuadrature<3, 8>::quadratures() {
+  Eigen::Matrix<double, 8, 3> quadratures;
+    quadratures(0, 0) = -1. / std::sqrt(3.);
+    quadratures(0, 1) = -1. / std::sqrt(3.);
+    quadratures(0, 2) = -1. / std::sqrt(3.);
+
+    quadratures(1, 0) = 1. / std::sqrt(3.);
+    quadratures(1, 1) = -1. / std::sqrt(3.);
+    quadratures(1, 2) = -1. / std::sqrt(3.);
+
+    quadratures(2, 0) = 1. / std::sqrt(3.);
+    quadratures(2, 1) = 1. / std::sqrt(3.);
+    quadratures(2, 2) = -1. / std::sqrt(3.);
+
+    quadratures(3, 0) = -1. / std::sqrt(3.);
+    quadratures(3, 1) = 1. / std::sqrt(3.);
+    quadratures(3, 2) = -1. / std::sqrt(3.);
+
+    quadratures(4, 0) = -1. / std::sqrt(3.);
+    quadratures(4, 1) = -1. / std::sqrt(3.);
+    quadratures(4, 2) = 1. / std::sqrt(3.);
+
+    quadratures(5, 0) = 1. / std::sqrt(3.);
+    quadratures(5, 1) = -1. / std::sqrt(3.);
+    quadratures(5, 2) = 1. / std::sqrt(3.);
+
+    quadratures(6, 0) = 1. / std::sqrt(3.);
+    quadratures(6, 1) = 1. / std::sqrt(3.);
+    quadratures(6, 2) = 1. / std::sqrt(3.);
+
+    quadratures(7, 0) = -1. / std::sqrt(3.);
+    quadratures(7, 1) = 1. / std::sqrt(3.);
+    quadratures(7, 2) = 1. / std::sqrt(3.);
+
+  return quadratures;
+}
+
+// Getting the weights for Tnquadratures = 8
+template <>
+inline Eigen::VectorXd mpm::HexahedronQuadrature<3, 8>::weights() {
+  Eigen::VectorXd weights(8);
+    weights(0) = 1.;
+    weights(1) = 1.;
+    weights(2) = 1.;
+    weights(3) = 1.;
+    weights(4) = 1.;
+    weights(5) = 1.;
+    weights(6) = 1.;
+    weights(7) = 1.;
+
+  return weights;
+}
+
+// Getting the quadratures for Tnquadratures = 27
+template <>
+inline Eigen::MatrixXd mpm::HexahedronQuadrature<3, 27>::quadratures() {
+  Eigen::Matrix<double, 27, 3> quadratures;
+    const double Qpoint_a = std::sqrt(3. / 5.);
```

Comment:
Hi @ezrayst could you please use a lower case for all variables, we no longer use capitalisation of `const` as `const`-ing is now very common than before. Thanks!

Reply:
Understood and changed!